libs/auth-nest-tools/src/lib/jwt.strategy.spec.ts

```diff
@@ -92,10 +93,13 @@ describe('JwtStrategy#validate', () => {
     }
     const request = ({
       headers: {
-        authorization: 'authorization',
+        authorization: testToken,
         'user-agent': 'test user agent',
         'x-forwarded-for': '2.2.2.2, 3.3.3.3',
       },
+      body: {
+        __accessToken: testToken,
+      },
```

Comment:
We should do this in a new test case to test this without the authorization header present.

Reply:
@thordurhhh  should we finish this one?

Reply:
Yes! I have updated this now